Add more instructions for installing OptiFine (#50)
PrismLauncher/prismlauncher.github.io: src/wiki/getting-started/installing-optifine.md

OptiFine will now be installed as a jarmod in that instance so you can remove the installer and extracted jarmod.

> From OptiFine Version **H1_pre2** the instructions for installing OptiFine on PrismLauncher have changed. Make sure to have the version of optifine that you wish to install, installed through the vanilla launcher.

Reviewer: ```suggestion
From OptiFine Version **H1_pre2** the instructions for installing OptiFine on Prism Launcher have changed. Make sure to have the version of optifine that you wish to install, installed through the vanilla launcher.
```

> From OptiFine Version **H1_pre2** the instructions for installing OptiFine on Prism Launcher have changed. Make sure to have the version of optifine that you wish to install, installed through the vanilla launcher.

Reviewer: ```suggestion
From OptiFine Version **H1_pre2** the instructions for installing OptiFine on Prism Launcher have changed. Make sure to have the version of Minecraft that you wish to install OptiFine on installed through the vanilla launcher.
```

1. Extract the MOD jar with the OptiFine installer
2. Open it with any archiver

> 3. Move all files from the folder called **notch** to the top folder of the archive file and click **save** 

Reviewer: ```suggestion
3. Move all files from the folder called **notch** to the top folder of the archive file and click **save**.
```

> 4. Move all files from the folder called **notch** to the root folder and click **save** (By root folder, I mean the top level folder of the archive file)

Reviewer: ```suggestion
3. Move all files from the folder called **notch** to the top folder of the archive file and click **save** 
```

> 4. Open Prism Launcher, edit the instance you wish to install OptiFine on, open the **Version** tab and click **Add to Minecraft.jar**, select the extracted / modfied OptiFine jarmod (the file ending in \_MOD.jar) and confirm.

Reviewer 2: modfied -> modified

Please run it through spellcheck next time

> 5. Open Prism Launcher, edit the instance you wish to install OptiFine on, open the **Version** tab and click **Add to Minecraft.jar**, select the extracted / modfied OptiFine jarmod (the file ending in \_MOD.jar) and confirm.

Reviewer: ```suggestion
4. Open Prism Launcher, edit the instance you wish to install OptiFine on, open the **Version** tab and click **Add to Minecraft.jar**, select the extracted / modfied OptiFine jarmod (the file ending in \_MOD.jar) and confirm.
```
